games-fps/alephone/alephone_marathon-20150907.recipe

```diff
@@ -0,0 +1,89 @@
+SUMMARY="Marathon 1 game assets"
+DESCRIPTION="Alien forces have boarded the interstellar colony ship \
+Marathon. The situation is dire. As a security officer onboard, it is your \
+duty to defend the ship and its crew.
+
+Experience the start of Bungie\'s iconic trilogy with Marathon. This release \
+uses the original Marathon data files for the most authentic experience \
+outside of a classic Mac or emulator."
+HOMEPAGE="https://alephone.lhowon.org/"
+COPYRIGHT="1991-2001 Bungie Software
+	Aleph One Developers"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://github.com/Aleph-One-Marathon/alephone/releases/download/release-20150620/Marathon-20150620-Data.zip"
```

Comment:
You might wish to use `$portVersion`. BTW, this will pull newer zip archive. (I did not test anything, so I don't know if the latest archive is fine.)
```diff
-SOURCE_URI="https://github.com/Aleph-One-Marathon/alephone/releases/download/release-20150620/Marathon-20150620-Data.zip"
+SOURCE_URI="https://github.com/Aleph-One-Marathon/alephone/releases/download/release-$portVersion/Marathon-$portVersion-Data.zip"
```

(Similar comments for `alephone_marathon2-20150907.recipe` and `alephone_marathoninfinity-20150907.recipe`.)